### ISSUE TITLE
[bug] Bot ignores label commands in the initial PR description

### ISSUE THREAD

**Issue by webknjaz** [ansible/ansibullbot#1003]
It looks like bot only takes into account commands, written in subsequent comments. I've seen people adding things like `+label: docsite_pr` right away on creation a few times now.

The last one I saw is: https://github.com/ansible/ansible/pull/44436

**Comment by mkrizek** [comment]
More like a feature request as https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md#how-to-use-label-commands explicitly mentions that label commands should be put into comments. Not sure what's the reason to not look into issue/PR description (maybe just that it wasn't implemented).

cc @jctanner

**Comment by webknjaz** [comment]
Even new, first-time contributors, do this: https://github.com/ansible/ansible/pull/44754#issue-211353673

**Comment by mkrizek** [comment]
Do you mean adding `+label: docsite_pr` even though it's not docsite pr?

**Comment by mkrizek** [comment]
Ah, nvm.

**Comment by webknjaz** [comment]
I've finally realized where this is coming from!

STR:
1. Go to https://docs.ansible.com/ansible
2. Click "Edit on GitHub" in top right corner
3. See
```
<!--- Your description here -->

+label: docsite_pr
```
in commit message field
4. After that it'll make it's way into PR as well (pre-filled from commit message)

**Comment by gundalow** [comment]
The `+label: docsite_pr`  was added so we can track who's using the "Edit on GitHub" link.

Would be great if the bot could honor this command in the PR description.

**Comment by webknjaz** [comment]
That was my guess, but I haven't found that in docs source. Is that configured somewhere separately?

**Comment by dagwieers** [comment]
Yeah, so the original reason for it to not work, was that the bot was not aware of the special "docsite_pr" label. We fixed that in https://github.com/ansible/ansibullbot/pull/975 together with some other improvements. But it seems that was insufficient (or as I suspected, the new code was not deployed yet).

**Comment by dagwieers** [comment]
@mkrizek So we need to make an exception for `docsite_pr` only if we would like this.

**Comment by dagwieers** [comment]
BTW We also added labels to GitHub, so that if you hover over a label, you get some meaningful information/instructions. We may want to improve/revisit that too.

@gundalow Tis also fits your Contributor Experience header !

**Comment by gundalow** [comment]
@webknjaz what does stable vs devel mean in this context?

So from reading the above:
1) `docsite_pr` is the only label that anyone can set
2) ONLY if it appears in the PR body 
3) And the PR body needs to be read, though only for this label

**Comment by webknjaz** [comment]
@gundalow there's `/stable/` and `/devel` addresses on the docs website. You could use different tags to know better where the user came from.

**Comment by gundalow** [comment]
Ah, the `latest` and `devel` links.
I don't think we need to worry about this at the moment. The link takes you to edit in devel, which is correct as PRs should always be done again `devel` the back ported.

Interesting idea, as the number of docs PRs increases we may want to do more filtering.
